source/common/http/conn_manager_impl.cc

```diff
@@ -322,11 +324,6 @@ Network::FilterStatus ConnectionManagerImpl::onData(Buffer::Instance& data, bool
     try {
       codec_->dispatch(data);
     } catch (const FrameFloodException& e) {
-      // TODO(mattklein123): This is an emergency substitute for the lack of connection level
```

Comment:
Can we delete the runtime guarded log below, any associated documentation, etc.?